grpc/service_config/service_config.proto

```diff
@@ -36,21 +36,32 @@ option java_outer_classname = "ServiceConfigProto";
 
 // Configuration for a method.
 message MethodConfig {
-  // The names of the methods to which this configuration applies. There must
-  // be at least one name. Each name entry must be unique across the entire
-  // ClientConfig. If the 'method' field is empty, then this MethodConfig
-  // specifies the defaults for all methods for the specified service.
+  // The names of the methods to which this configuration applies.
+  // - MethodConfig without names (empty list) will be skipped.
+  // - Each name entry must be unique across the entire ClientConfig.
+  // - If the 'method' field is empty, this MethodConfig specifies the defaults
+  //   for all methods for the specified service.
+  // - If the 'service' field is empty, the 'method' field must be empty, and
+  //   this MethodConfig specifies the default for all methods (it's the default
+  //   config).
   //
   // For example, let's say that the service config contains the following
```

Comment:
Right before this sentence, let's explicitly add the sentence "When determining which MethodConfig to use for a given RPC, the most specific match wins."

Reply:
Done